### PR TITLE
Add a default geojson for area

### PR DESCRIPTION
**Pull request by RemiDesgrange**
Connected to https://github.com/georchestra/georchestra/pull/3028

Need to be backported to `docker-master` too.

### CODE REVIEW

File: console/console.properties [hunk 1]
```diff
@@ -146,23 +146,22 @@ orgTypeValues=Association,Company,NGO,Individual,Other
...
 # Center of map
-AreaMapCenter=2.5, 50
+AreaMapCenter=9.3707, 42.0753
 
 # Zoom of map
-AreaMapZoom=6
+AreaMapZoom=7
 
 # AreasUrl is the URL of a static file or a service with a GeoJSON FeatureCollection object string in EPSG:4326.
 # example "dynamic" AreasUrl=https://my.server.org/geoserver/ows?SERVICE=WFS&REQUEST=GetFeature&typeName=gadm:gadm_for_countries&outputFormat=json&srs=EPSG:4326&cql_filter=ISO='FRA' or ISO='BEL'
```
Comment (fvanderbiest) [inline console/console.properties:156]: Missing a comment explaining that the file will also be searched for in the `<datadir>/console/` folder

File: console/console.properties [hunk 2]
```diff
@@ -146,23 +146,22 @@ orgTypeValues=Association,Company,NGO,Individual,Other
...
-AreasUrl=https://www.geo2france.fr/public/communes_zones_competence.geojson
+AreasUrl=area.geojson
 
 # The following properties are used to configure the map widget behavior:
 # AreasKey is the key stored in the org LDAP record to uniquely identify a feature.
 AreasKey=INSEE_COM
 
 # AreasValue is the feature "nice name" which appears in the widget list once selected, and in the search result as well.
-AreasValue=NOM_COM_IN
+AreasValue=NOM_COM_N
```
Comment (fvanderbiest) [inline console/console.properties:164]: "NOM_COM_M" if I trust the above file you submitted.

File: console/area.geojson [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+{ "type": "Feature", "properties": { "ID": "COMMUNE_0000000009763028", "NOM_COM_M": "LAVATOGGIO", "INSEE_COM": "2B138", "INSEE_CAN": "07", "INSEE_ARR": "5", "INSEE_DEP": "2B" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 8.877, 42.561 ], [ 8.877, 42.562 ], [ 8.875, 42.565 ], [ 8.879, 42.568 ], [ 8.88, 42.568 ], [ 8.883, 42.57 ], [ 8.885, 42.572 ], [ 8.884, 42.576 ], [ 8.881, 42.576 ], [ 8.88, 42.578 ], [ 8.877, 42.577 ], [ 8.873, 42.578 ], [ 8.871, 42.58 ], [ 8.868, 42.582 ], [ 8.865, 42.582 ], [ 8.863, 42.583 ], [ 8.86, 42.583 ],
```
Comment (fvanderbiest) [inline console/area.geojson:363]: I think you can filter out the `ID`, `INSEE_CAN`, `INSEE_ARR` fields which are useless and make the response heavier.

**Review by fvanderbiest — CHANGES_REQUESTED** [review]

**Comment by landryb** [comment]
So in the end it was decided to ship corsica as default, after pointing for a long time at a picardie geojson url ? how.. fun :)

**Comment by RemiDesgrange** [comment]
I had a preference for Limousin since a lot of persons, even in France, don't know where it is, but it doesn't exist anymore :-)

**Pull request merged**